src/main/java/com/couchbase/lite/replicator/PullerInternal.java

```diff
@@ -700,18 +720,29 @@ public void onCompletion(Response httpResponse, Object result, Throwable e) {
                                 setError(e);
                             }
                         } else {
-                            Map<String, Object> properties = (Map<String, Object>) result;
+                            Map<String, Object> properties = (Map<String, Object>) contentBody;
                             PulledRevision gotRev = new PulledRevision(properties);
```

Comment:
this should be `PulledRevision gotRev = new PulledRevision(properties, contentSize);`